src/dotnet/devcontainer-feature.json

```diff
@@ -1,30 +1,30 @@
 {
     "id": "dotnet",
-    "version": "1.1.3",
-    "name": "Dotnet CLI",
+    "version": "2.0",
+    "name": ".NET",
```

Comment:
I'll defer all naming/description changes to you and @samruddhikhandale. I copied [Microsoft's .NET branding](https://dotnet.microsoft.com/en-us/download), but I know it can be disruptive to rename things.